### ISSUE TITLE
Remove inline Tip button tooltips on hover

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#28699]
## Description

Due to bugs and inconsistency with how these tooltips can appear, we will remove them.

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
Brave | 1.56.20 Chromium: 115.0.5790.171 (Official Build) (64-bit)
-- | --
Revision | cf9067bf10d8f798c24643029af1d24e275646d6-refs/branch-heads/5790@{#1924}
OS | Windows 10 Version 22H2 (Build 19045.3208)
```

Verified using dev component for:

```
Brave Local Data Updater - Version: 1.0.942
```
![image](https://github.com/brave/brave-browser/assets/38657976/31f7e2e5-f11b-47e5-87c6-b71fc4d0aa55)

Launched `1.56.20` using `--use-dev-goupdater-url` and ensured issue is fixed with the component `1.0.942`. Confirmed that there is no inline Tip button tooltip for the Twitter and Reddit publishers

https://twitter.com/brave
https://twitter.com/bravelaurenwags
http://reddit.com/u/beachbum0407
https://www.reddit.com/u/kjozwiak

Example | Example | Example | Example
---------|-----------|-----------|---------
![image](https://github.com/brave/brave-browser/assets/38657976/3b53d4d5-cf00-4bb8-9353-476fdfda3fc5) | ![image](https://github.com/brave/brave-browser/assets/38657976/5eca8ab7-9457-4687-8994-f5da8465a496) | ![image](https://github.com/brave/brave-browser/assets/38657976/dedec34b-6226-4f38-bbf5-2139e8dbe1ee) | ![image](https://github.com/brave/brave-browser/assets/38657976/bf975d8f-8b63-4720-a2a2-d603547cc42b)

**Comment by emerick** [comment]
This is in Greaselion `1.0.408`.
